### DISCUSSION TITLE
h5oina file

### DISCUSSION THREAD

**Discussion by ArulUQ** [mtex-toolbox/mtex#1116] [Ask Anything]
Hi, 
I could not able to import H5OINA file type!
Do i have to convert it to crc/cpr or any suggestion, please?
Thanks

**Comment by nyyssont** [comment]
Converting h5oina to either crc/cpr or ctf should work.

**Reply by kilir** [reply]
Can you please test the h5oina interface and report if  this works, if not what error is produced. H5oina should be preferred over cpr/crc or ctf, hoever my collection of examples is limited.

See loadEBSD_h5oina in the development branch.

Cheers,
Rüdiger

**Reply by ArulUQ** [reply]
Thanks Nyyssonen. It works as usual after converting into CPR.

**Reply by ArulUQ** [reply]
Thanks Rudiger as i tried loadEBSD_h5oina but unsuccessful due to the compatibility issues but when i converted into CPR file which works fine as usual :)

**Reply by kilir** [reply]
Do you mind sharing the error message, Aztec version and a description of you data?

**Reply by kilir** [reply]
It could also be that your hdf5 file is version > 1.8 (which is what Matlab can handle in the moment).  h5oina is version 1.10. Using h5format_convert from the hdf5 tools allows the fiel to be readable by Matlab.
Rüdiger
